Gems/Atom/Feature/Common/Assets/ShaderResourceGroups/ViewSrg.azsli

```diff
@@ -25,6 +25,7 @@ partial ShaderResourceGroup ViewSrg
     float3 m_worldPosition;
 
     float4 m_nearZ_farZ_farZTimesNearZ_farZMinusNearZ;
```

Comment:
Oops. Forgot to remove this. Will address it for my next commit when addressing any other feedback